packages/paste-website/src/components/tokens-list/token-card/token-example/SpacingExample.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {useTheme, remToPx} from '@twilio-paste/theme';
+
+interface SpacingExampleProps {
+  tokenName: string;
+  spacing: string;
+}
+
+export const SpacingExample: React.FC<SpacingExampleProps> = ({tokenName, spacing}) => {
+  if (spacing === '0') return null;
+
+  const theme = useTheme();
+  const lineColor = theme.borderColors.colorBorder;
+  const lineStart = 1;
+  const spacingPx = Math.abs(remToPx(spacing) as number);
+  const lineEnd = lineStart + spacingPx;
+  const svgWidth = lineEnd + 2;
+  const strokeDash = tokenName.toLowerCase().includes('negative') ? 4 : 1;
+
+  return (
+    <svg
+      width={`${svgWidth}px`}
+      height="64px"
+      viewBox={`-1 0 ${svgWidth} 64`}
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
```

Comment:
suggestion (non-blocking): I _think_ we don't need these for SVG in React.

Reply:
It seems fine either way, but most of the "SVG in React" docs I see online have these attrs set, and there's not a compelling reason to _not_ include them (they're valid React props) so I'm inclined to just let them be?